### PR TITLE
ACI dynamic compilation & kryo deserialization

### PR DESCRIPTION
**Pull request by tbekas**

### CODE REVIEW

File: src/main/scala/org/tessellation/aci/ACIRepository.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                 t.id, t.address, t.definition
+          from aci_mapping m 
+          join aci_type t
+          on m.dst_type_id = t.id
+         where m.src_type_id = $srcTypeId"""
+      .query[(ACITypeMapping, ACIType)]
+      .to[List]
+      .transact(xa)
+
+  def saveAciType(aciType: ACIType): F[Int] =
```
Comment (buckysballs) [inline src/main/scala/org/tessellation/aci/ACIRepository.scala:32]: Here (or in the endpoint) we will want to do a validation step which checks if the type inherits from Omega

Comment (buckysballs) [reply src/main/scala/org/tessellation/aci/ACIRepository.scala:32]: is it of type A <: \Omega

Comment (kpudlik) [reply src/main/scala/org/tessellation/aci/ACIRepository.scala:32]: Ω

File: src/main/scala/org/tessellation/aci/ExplicitKryoRegistrar.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    classes.foreach {
+      case (registryClass, id) => k.register(registryClass, id)
+    }
+}
+
+object ExplicitKryoRegistrar {
+  implicit val monoid: Monoid[ExplicitKryoRegistrar] =
+    new Monoid[ExplicitKryoRegistrar] {
+
+      def combine(x: ExplicitKryoRegistrar,
```
Comment (buckysballs) [inline src/main/scala/org/tessellation/aci/ExplicitKryoRegistrar.scala:19]: When we are recompiling the registrar, we want to include all algebras/coalgebras and implicit methods into the Home object

**Comment by tbekas** [comment]
Closing in favour of a new PR

**Pull request closed without merging**